test/e2e/storage/persistent_volumes.go

```diff
@@ -38,6 +40,11 @@ import (
 	imageutils "k8s.io/kubernetes/test/utils/image"
 )
 
+const (
+	// pvDeletionInTreeProtectionFinalizer is the finalizer added to protect PV deletion for in-tree volumes.
+	pvDeletionInTreeProtectionFinalizer = "kubernetes.io/pv-controller"
```

Comment:
Can you use the existing constant for this finalizer rather than define a new constant?
https://github.com/kubernetes/kubernetes/blob/v1.24.0-alpha.4/pkg/controller/volume/persistentvolume/util/util.go#L83

Reply:
![image](https://user-images.githubusercontent.com/13997135/160224454-814f132a-b6e2-47d5-852e-0f7224a67829.png)
if we use this imports, there is some errors like this.